### ISSUE TITLE
Unable to use URLSession.shared.dataTask in v14.0.0

### ISSUE THREAD

**Issue by tayathorn** [facebook/facebook-ios-sdk#2082]
### Checklist before submitting a bug report

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](https://github.com/facebook/facebook-ios-sdk/blob/main/CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

### Xcode version

13.2.1

### Facebook iOS SDK version

14.0.0

### Dependency Manager

SPM

### SDK Framework

Core

### Goals

I want standard method: `URLSession.shared. dataTask()` works normally

### Expected results

`URLSession.shared. dataTask()` works fine in the previous version, 13.2.0, but breaks when updated to version 14.0.0.

### Actual results

Got an error `Cannot convert value of type 'URLRequest' to expected argument type 'URL'`
<img width="1346" alt="Screen Shot 2565-06-20 at 11 56 26" src="https://user-images.githubusercontent.com/31066087/174528068-731ce6c4-96f2-4ac8-9462-9d17a5b744eb.png">

### Code samples & details

```swift
// INSERT YOUR CODE HERE
func fetch(urlRequest: URLRequest, completion: @escaping (Result<Data, NetworkError>)->()) -> URLSessionDataTask {
        let urlRequest = URLRequest(url: URL(string: "Any API endpoint")!)
        let task = URLSession.shared.dataTask(with: urlRequest1) { (data, response, error) in
            print("== FETCH ==")
        }
        return task
    }
```

**Comment by samodom** [comment]
@tayathorn This is curious behavior since somehow the method exposed by URLSession is hidden by the protocol method declared by us.  We will come up with a solution for this and, in the meantime, feel free to continue using a prior version of the SDK that doesn't have this issue.

**Comment by mattiacantalu** [comment]
We had the same issue, extending the `URLSession` with a custom protocol, like:
```
extension URLSession: URLSessionProtocol {}
```

For now, we are unable to update to latest version. Hope a fix soon.

**Comment by joesus** [comment]
Fix is released with 14.1.0. Please let us know if this does not resolve the issue. Thank you for your patience!
